### PR TITLE
Notion/store locally in avro

### PR DESCRIPTION
**Pull request by teopopescu**
This PR contains several functions for taking a CSV file containing ZRX data and converting it into avro format based on the price_schema.avsc schema

### CODE REVIEW

File: data_extraction.log [hunk 1]
```diff
@@ -0,0 +1,68 @@
+DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): min-api.cryptocompare.com:443
```
Comment (ankitchiplunkar) [inline data_extraction.log]: can we remove this file?

File: test.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from price_pipeline import price_pipeline
+#from transform_data import transform_data
+
+test = price_pipeline()
+#test.extract_historical_data('ZRX',1502928000)
+
+test.transform_data('ZRX')
```
Comment (ankitchiplunkar) [inline test.py]: I don't see this function in the price_pipeline class?
Is this test passing?

File: test_ZRX.csv [hunk 3]
```diff
@@ -0,0 +1,14019 @@
+,close,high,low,open,time,volumefrom,volumeto,pair
```
Comment (sp6pe) [inline test_ZRX.csv]: csv should not be in the pr

File: transform_data.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import avro.schema
+from avro.datafile import DataFileReader, DataFileWriter
+from avro.io import DatumReader, DatumWriter
+import csv
+from collections import namedtuple
+
+#class transform_data():
```
Comment (sp6pe) [inline transform_data.py]: remove if not used

**Pull request merged**